app/app.js

```diff
@@ -1384,6 +1384,30 @@ var AppComponent = React.createClass({
     }
   },
 
+  handleActionableError: function(error, message, link) {
+
+    var utcTime = usrMessages.MSG_UTC + new Date().toISOString();
+
+    message = message || '';
+    //send it quick
+    app.api.errors.log(this.stringifyErrorData(error), message, '');
+
+    var body = (
+      <div>
+        <p>{message}</p>
+        {link}
+      </div>
+    );
+
+    this.setState({
+      notification: {
+        type: 'alert',
+        body: body,
+        isDismissable: true
```

Comment:
I defnitely think it should remain dismissable. Could we also auto-dismiss it once your set-up is successful? It's not a great experience to still have the error there after you finish.